README.md

```diff
@@ -12,23 +12,26 @@ Once `esy` is available, run
 esy
 ```
 
-to install all dependencies.
+to install all dependencies and build the project.
 
-Then:
+When run, `esy` will (as specified in `esy.json`):
 
-```bash
-esy build
-```
-
-to build the project. This command, as constructed in `esy.json`,
-
-1. generates a symbolically linked dir `bs-platform` in `node_modules`, which is helpful when using JavaScript bundlers like Webpack that expect to find Melange stdlib JavaScript files under `node_modules/bs-platform`.
-2. builds the project.
+1. Generate a symbolically linked dir `bs-platform` in `node_modules`, which is
```

Comment:
This should be `melange` on the latest commit. Might need to be updated in the `esy.json` file.

Reply:
```suggestion
1. Generate a symbolically linked dir `melange` in `node_modules`, which is
```

Reply:
Good catch!